tablo/models.py

```diff
@@ -86,6 +86,24 @@ def finalize(self, dataset_id):
                 new_table_name=new_table_name
             ))
 
+            c.execute('SELECT sequencename FROM pg_sequences WHERE sequencename LIKE \'%{}%\''.format(old_table_name))
+            for (old_sequence_name,) in c.fetchall():
```

Comment:
Why do we read in a `tuple`? Do we just need the first argument from `c.fetchall()`? I've never seen this syntax.

Reply:
Each row of `c.fetchall()` has only one item in it, so this syntax expands the row and puts the first item in `old_sequence_name`.